src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.skia.cs

```diff
@@ -14,14 +12,13 @@ internal class TextBoxView
 	{
 		private readonly IOverlayTextBoxViewExtension? _textBoxExtension;
 
-		private readonly WeakReference<TextBox> _textBox;
 		private readonly bool _isPasswordBox;
 		private bool _isPasswordRevealed;
 		private readonly bool _isSkiaTextBox = !FeatureConfiguration.TextBox.UseOverlayOnSkia;
 
 		public TextBoxView(TextBox textBox)
 		{
-			_textBox = new WeakReference<TextBox>(textBox);
+			TextBox = textBox;
```

Comment:
Based on Jerome's comment on the other thread, I understand we should use the `WeakReferencePool` instead of the `WeakReference`